be/HeatMapProfiler.py

```diff
@@ -129,17 +133,27 @@ def wrapper(*args, **kwargs):
     args, argv = parser.parse_known_args()
 
     if args.display:
-        print('display')
-        # TODO: Copy given file to default name/location
-        pass
+        print(f'Displaying {args.display}')
+        if not args.display.endswith(".json"):
+            print(f"Warning: displaying a non-JSON file: {args.display}")
+        copyfile(args.display, DEAFAULT_FILEPATH)
     elif not args.view and args.target:
         profiler = HeatMapProfiler(
             sample_mode=args.sample, thread_mode=args.thread, period=args.period)
         profiler.run_file(args.target, [args.target] + argv)
 
     if args.export:
-        print('export')
-        # TODO: Copy default file to given location with new name (append timestamp)
-        pass
-
-    # TODO: invoke frontend here
+        print(f'Exporting to {args.export}')
+        import datetime
+        if os.path.isdir(args.export):
+            copyfile(DEAFAULT_FILEPATH, args.export + f'line_level_profile{datetime.datetime.now().isoformat()}.json')
+        else:
+            if os.path.exists(args.export):
+                print(f"Warning: overwriting existing file: {args.export}")
+            if not args.export.endswith(".json"):
+                print(f"Warning: writing to a non-JSON format: {args.export.split('.')[-1]}")
```

Comment:
I'm being really picky, but can we change "format" to "file", since we will still be writing in JSON format, it's just the file extension is wrong.